docker-compose-rule-core/src/main/java/com/palantir/docker/compose/CustomImmutablesStyle.java

```diff
@@ -20,5 +20,5 @@
 import org.immutables.value.Value.Style;
 
 @Target({ElementType.PACKAGE, ElementType.TYPE})
-@Style(depluralize = true, strictBuilder = true)
+@Style(depluralize = true, strictBuilder = true, overshadowImplementation = true)
```

Comment:
This is also technically a break as people may have done `ImmutableDockerComposeRule dcr = DockerComposeRule.builder()....build()` in the past, which is now a `DockerComposeRule`. A few people have done this: https://github.com/search?q=ImmutableDockerComposeRule&type=Code, and 1 further case internally, but that's it. I need the later behaviour in order to instrument the DCR class properly.